charts/incubator/radicale/templates/_configmap.tpl

```diff
@@ -149,7 +149,7 @@ data:
 
     # Don't include passwords in logs
     #mask_passwords = True
-    mask_passwords = {{ .Values.radicale.logging.mask_passwords }}
+    mask_passwords = {{ .Values.radicale.logging.mask_passwords | ternary "True" "False" }}
```

Comment:
I remember I had to escape `"` at some point. But this was in `values.yaml` Maybe you don't have that problem in the configmap. Just keep it mind when you are testing live! And if needed come back to fix it with another PR ;)

Reply:
Already tested this in my debugging helper chart. Escaping caused: `unexpected "\\" in operand`